tests/unit/src/main/scala/tests/BaseLspSuite.scala

```diff
@@ -87,18 +87,30 @@ abstract class BaseLspSuite(
     assertNoDiff(obtained, expectedName)
   }
 
-  def test(testOpts: TestOptions, withoutVirtualDocs: Boolean)(
+  def test(
+      testOpts: TestOptions,
+      withoutVirtualDocs: Boolean,
+      maxRetry: Int = 0,
```

Comment:
Added a retry to use in FileWatcheLspSuite, since it's flaky on MacOS and I am not sure if that is something that we can fix. Last time I tried I failed to fix anything there.